Remove global karma option in autotest-js.sh

### Steps to reproduce
1. install karma-cli globally on your system (`npm install -g karma-cli`)
2. execute `./autotest-js.sh`

### Expected behaviour
autotest-js.sh runs just fine

### Actual behaviour
autotest-js.sh crashes after installation of node_modules:
```
Error: Cannot find module 'build/node_modules/karma/lib/cli'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at requireCliAndRun (/usr/local/lib/node_modules/karma-cli/bin/karma:41:20)
    at /usr/local/lib/node_modules/karma-cli/bin/karma:78:14
    at /usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:46:14
    at process (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:69:43)
```

------

karma is installed locally with autotest-js.sh anyway, so this instance should always be used.
So let's simply skip `KARMA="$(which karma 2>/dev/null)"`

--- a/autotest-js.sh
+++ b/autotest-js.sh
@@ -22,19 +22,7 @@ fi
 # update/install test packages
 mkdir -p "$PREFIX" && $NPM install --link --prefix "$PREFIX" || exit 3
 
-KARMA="$(which karma 2>/dev/null)"
-
-# If not installed globally, try local version
-if test -z "$KARMA"
-then
-	KARMA="$PREFIX/node_modules/karma/bin/karma"
-fi
-
-if test -z "$KARMA"
-then
-	echo 'Karma module executable not found' >&2
-	exit 2
-fi
+KARMA="$PREFIX/node_modules/karma/bin/karma"
 
 NODE_PATH='build/node_modules' KARMA_TESTSUITE="$1" $KARMA start tests/karma.config.js --single-run
 

